src/main/webapp/join-us.html

```diff
@@ -203,11 +203,7 @@ <h5 class="mb-0">
                     <div class="display-3 font-weight-light text-darker">Join the <span
                             class="font-weight-bold d-inline">Team</span></div>
                     <p class="lead text-darker mt-auto mb-auto">I want to join the SEF Team</p>
-                    <a class="btn btn-purple text-uppercase mt-3"
-                       href="apply.html"
-                    >
-                        Apply now!
-                    </a>
+                    <a href="apply.html"><button class="btn btn-purple text-uppercase mt-3" href="apply.html">Apply now!</button></a>
```

Comment:
It's better if we can indent the code like this  -->
```
<a href="apply.html">
     <button class="btn btn-purple text-uppercase mt-3">
          Apply now!
     </button>
</a>
```

Reply:
Also we can remove the href attribute in the button

Reply:
Also we can remove the `href` attribute in the button tag

Reply:
No problem, I'll do that :) !
Actually, it looks like a <button> shouldn't be nested in a <a href>. If you try to validate the HTML code in https://validator.w3.org for example, it will tell you that.

Reply:
> Also we can remove the `href` attribute in the button tag

Yep, already did for this button, but we should do it for the other indeed, and/or do what I told you in my previous comment ?

Reply:
Oh right. Then let's try adding `!important` rule for the color of "btn-purple" class. So we can keep the join-us.html file same as before

Reply:
Hi mate,
What I meant is this : https://stackoverflow.com/questions/2906582/how-to-create-an-html-button-that-acts-like-a-link

Either the HTML way (with form) or CSS way (create a "a" styled as a button with css). Tell me what you preefer and I'll do it ;)

Reply:
Hello @AlexARNcode ,
Let's keep only the button tag and place the href as a prop of the button instead of wrapping the button in an `<a>` or adding CSS to the `<a>` to look like button.